pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -876,7 +876,7 @@ Block.Tabular:
           default: TABULAR
           enum:
             - TABULAR
-        numberOfColumns:
+        columnsCount:
```

Comment:
Changed the property name according to the line 870.

Reply:
Oh! I kind of like numberOfColumns better! I think I introduced the mistake. If you feel really strongly about keeping columnsCount, the fine. Otherwise then let's fix the `required` entries instead.